tests/test_bandpass.py

```diff
@@ -309,7 +301,7 @@ def test_zp():
     """Check that the zero points are maintained in an appropriate way when thinning, truncating."""
     # Make a bandpass and set an AB zeropoint.
     bp = galsim.Bandpass(os.path.join(datapath, 'LSST_r.dat'), 'nm')
-    bp = bp.withZeropoint(zeropoint='AB', effective_diameter=6.4, exptime=15)
```

Comment:
I don't see any new unit tests to replace the old way of setting diameter and exptime.  There are tests of drawImage with area, exptime for regular GSObjects, but I don't think there are any for Chromatic ones.

Possibly, this could go into the check_chromatic_invariant helper function.